src/polar/PolarRadiusAxis.tsx

```diff
@@ -9,11 +9,6 @@ import Layer from '../container/Layer';
 import { polarToCartesian } from '../util/PolarUtils';
 import { PresentationAttributes, filterProps, BaseAxisProps, TickItem, adaptEventsOfChild } from '../util/types';
 
-interface TickIem {
```

Comment:
Unused. The correct `TickItem` is already being imported, so we can just delete this one.